Lib/test/test_io/test_bufferedio.py

```diff
@@ -962,6 +962,16 @@ def test_args_error(self):
         with self.assertRaisesRegex(TypeError, "BufferedWriter"):
             self.tp(self.BytesIO(), 1024, 1024, 1024)
 
+    def test_closed_errors(self):
```

Comment:
I think this better describes the test:
```suggestion
    def test_non_boolean_closed_attr(self):
```